hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -327,3 +327,48 @@ def test_manual_remove_element(self):
         assert (
             sorted(s._xray_markers.keys()) ==
             ['Zn_Ka', 'Zn_La'])
+
+
+@lazifyTestClass
```

Comment:
Ok, I did try it manually to load some data lazily and rebin the data and if I remember correctly, it did rebin the data, so I naively assumed that it was working... but I may have done something wrong or miss something!
Agree, better to remote it for now.

Are the tests still supposed to pass after the signals have been lazified inside the tests?